code/pcgen-launch4j.cfg.xml

```diff
@@ -1,23 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <launch4jConfig>
   <dontWrapJar>true</dontWrapJar>
   <headerType>gui</headerType>
   <jar>pcgen.jar</jar>
-  <outfile>C:\Projects\pcgendev\code\pcgen.exe</outfile>
+  <outfile>Z:\Netbeans\pcgen\code\pcgen.exe</outfile>
   <errTitle></errTitle>
   <cmdLine></cmdLine>
   <chdir></chdir>
   <priority>normal</priority>
   <downloadUrl>http://java.com/download</downloadUrl>
   <supportUrl></supportUrl>
-  <customProcName>false</customProcName>
   <stayAlive>false</stayAlive>
+  <restartOnCrash>false</restartOnCrash>
   <manifest></manifest>
-  <icon>C:\Projects\pcgen-release\pcgen\installers\win-installer\Local\pcgen.ico</icon>
+  <icon>Z:\Netbeans\pcgen\installers\win-installer\Local\pcgen.ico</icon>
   <jre>
-    <path></path>
-    <minVersion>1.6.0</minVersion>
-    <maxVersion>1.7.9</maxVersion>
+    <path>jre/</path>
+    <bundledJre64Bit>false</bundledJre64Bit>
+    <bundledJreAsFallback>false</bundledJreAsFallback>
+    <minVersion>1.8.0</minVersion>
+    <maxVersion>1.8.9</maxVersion>
```

Comment:
Version set to 1.8 is premature (project still using 1.7 until @cpmeister updates code base to accept 1.8)